flask_admin/model/base.py

```diff
@@ -1242,11 +1291,45 @@ def _create_ajax_loader(self, name, options):
         raise NotImplementedError()
 
     # Views
-    @expose('/')
+    @expose('/', methods=('POST', 'GET'))
```

Comment:
Here's the new endpoint: https://github.com/mrjoes/flask-admin/pull/756/files#diff-401640c72e6a6b2a48e711090af0534cR1494